### PR TITLE
chore: bump dependencies

### PR DESCRIPTION
**Pull request by smira**
| Package | Update | Change |
|---|---|---|
| [curl/curl](https://togithub.com/curl/curl) | minor | `8_0_1` -> `8_1_2` | 
| [davea42/libdwarf-code](https://togithub.com/davea42/libdwarf-code) | minor | `0.6.0` -> `0.7.0` | 
| git://git.kernel.org/pub/scm/git/git.git | minor | `2.40.1` -> `2.41.0` | 
| git://git.kernel.org/pub/scm/utils/util-linux/util-linux.git | minor | `2.38.1` -> `2.39` | 
| git://git.openssl.org/openssl.git | patch | `1_1_1t` -> `1_1_1u` | 
| git://git.savannah.gnu.org/diffutils.git | minor | `3.9` -> `3.10` | 
| [https://git.lysator.liu.se/nettle/nettle.git](https://git.lysator.liu.se/nettle/nettle) | minor | `3.8.1` -> `3.9.1` | 
| [https://gitlab.kitware.com/cmake/cmake.git](https://gitlab.kitware.com/cmake/cmake) | patch | `3.26.3` -> `3.26.4` | 
| [libuv/libuv](https://togithub.com/libuv/libuv) | minor | `v1.44.2` -> `v1.45.0` | 
| [mesonbuild/meson](https://togithub.com/mesonbuild/meson) | patch | `1.1.0` -> `1.1.1` | 
| [protocolbuffers/protobuf](https://togithub.com/protocolbuffers/protobuf) | minor | `23.0` -> `23.2` |

### CODE REVIEW

File: openssl/pkg.yaml [hunk 1]
```diff
@@ -45,6 +45,6 @@ steps:
       - |
         cd openssl
         make DESTDIR=/rootfs install_sw
-finalize:
+finalize:curl -
```
Comment (frezbo) [inline openssl/pkg.yaml:48]: typo?

**Review by frezbo — APPROVED** [review]

File: openssl/pkg.yaml (hunk 1 above)
Comment (smira) [reply openssl/pkg.yaml]: already removed, thanks!

**Pull request merged**